backend/apid/routers/users.go

```diff
@@ -59,21 +59,6 @@ func (r *UsersRouter) Mount(parent *mux.Router) {
 	routes.Path("{id}/{subresource:password}", r.updatePassword).Methods(http.MethodPut)
 }
 
-func (r *UsersRouter) list(w http.ResponseWriter, req *http.Request) (interface{}, error) {
-	records, continueToken, err := r.controller.Query(req.Context())
-
-	// Obfustace users password
-	for i := range records {
-		records[i].Password = ""
```

Comment:
The generic List doesn't do that for Users. Is that a problem?

Reply:
Oh god yes, good catch, that's actually a real problem haha. I'll move that to the controller actually!